### PR TITLE
Added basic component-registry loader

### PR DESCRIPTION
**Pull request by fernandojsg**
Based on the latest changes from https://github.com/aframevr/aframe-registry
screen shot 2016-09-25 at 16 14 16

### CODE REVIEW

**Comment by fernandojsg** [comment]
I'm not sure about which name to use for grouping the components. At first I thought about "core" and "registry/community" but if they user already has some custom components loaded they'll appear on "core" too as it shows really the names of the components already loaded.
Any thoughts on that @ngokevin ?

**Comment by ngokevin** [comment]
For now, it could just be `registered` + `community` or `registry`. 

Though you can also check components against the `registry` to determine whether or not they are core, developer-registered, or community.

But not too important since the dropdown is just temporary right?

**Comment by fernandojsg** [comment]
I've just changed it to `registered` + `registry`. The dropdown as it's right now it will be just temporary but still we'll have a autocomplete dropdown where you can access these components, as discussed on https://github.com/aframevr/aframe-inspector/issues/317: https://cloud.githubusercontent.com/assets/782511/17860914/6b424410-688f-11e6-9c73-b27bc81d59af.gif

Next to this dropdown you'll have a components button that will popup the big dialog similar to Unity's store. You'll use that dialog when you want to know more info about a component, or when you want to discover new components as it will show some descriptions, links to author/components, tags, and so on.
But still we want to keep the autocomplete-dropdown as it's faster to include already known and commonly used components such as `geometry`, `material` and so on.

File: src/components/components/AddComponent.js [hunk 1]
```diff
@@ -13,39 +14,68 @@ export default class AddComponent extends React.Component {
...
+      Events.emit('componentAdded', {entity: entity, component: componentName});
+      ga('send', 'event', 'Components', 'addComponent', componentName);
     }
 
-    entity.setAttribute(componentName, '');
-    Events.emit('componentAdded', {entity: entity, component: componentName});
-    ga('send', 'event', 'Components', 'addComponent', componentName);
+    if (origin === 'community') {
+      var packageName = selectedOption.value;
+      INSPECTOR.componentLoader.addComponentToScene(packageName, function (componentName) {
```
Comment (ngokevin) [inline src/components/components/AddComponent.js]: can one line `INSPECTOR.componentLoader.addComponentToScene(packageName, addComponent)`

File: src/lib/componentloader.js [hunk 2]
```diff
@@ -7,29 +7,29 @@ ComponentLoader.prototype = {
   loadComponentsData: function () {
     var xhr = new window.XMLHttpRequest();
     // @todo Remove the sync call and use a callback
-    xhr.open('GET', 'https://raw.githubusercontent.com/aframevr/aframe-components/master/components.json', false);
+    xhr.open('GET', 'https://aframe.io/aframe-registry/build/' + AFRAME.version + '.json');
```
Comment (ngokevin) [inline src/lib/componentloader.js]: you have to restrict this to the major version:
- if it's 0.3.x, use 0.3.0
- if it's 4.x.x, use 4.0.0

**Review by ngokevin — APPROVED** [review]

File: src/components/components/AddComponent.js (hunk 1 above)
Comment (fernandojsg) [reply src/components/components/AddComponent.js]: Yep! I missed it :D

File: src/lib/componentloader.js (hunk 2 above)
Comment (fernandojsg) [reply src/lib/componentloader.js]: I've just added the function `getMajorVersion (version)` that takes cares of that

**Pull request merged**